### ISSUE TITLE
Intermittent sound on Chrome

### ISSUE THREAD

**Issue by bflora2** [goldfire/howler.js#611]
I'm seeing an odd problem with Howler in Chrome (PC): My game sets up and plays sound effects as follows:

```
// Each sound is created once...
//
var sound1 = new Howl({
    src: ['sound1.webm', 'sound1.mp3'],
    volume: 0.3
});

// ... later, the sound may be played multiple times during the game.
//
sound1.play();
```

The problem is that on very infrequent occasions one or more of the created sound effects will not play (i.e., the sound effect is silent when played). When a particular sound effect does not play it never plays during that game session (the page must be reloaded to regain the sound). Both webm and mp3 sound effects are affected. The browser's network panel always shows all sound files were successfully downloaded by Howler. There is no code in the game that mutes, stops, or changes the volume of the sound effects.

When a sound does not play ```sound.play()``` still returns a valid looking sound id, and the "end" event is still triggered (i.e., if the sound is 1 second in duration then the "end" event is triggered after one second). So, it is as if Howler thinks everything is ok. This makes me wonder if the problem is actually in Chrome. Note that the little speaker icon in the browser tab does not appear when an attempt is made to play the broken sound.

I do not know if browsers other than Chrome are affected since the problem is so infrequent (I see the problem maybe once every day or two). I have had a Chrome user report the exact same problem, so I know it is not just a quirk on my machine. At his point, I've switched from ```howler.min.js``` to ```howler.js``` so I can use the debugger to dive into the Howler code the next time I see the problem happen and try learn more about what is going on.

Meanwhile, I want to ask, has anyone else seen this problem? Do you have any Howler debugging tips I can use to help track this issue down?

**Comment by bflora2** [comment]
I forgot to mention... I using the Howler 2.0.0

The problem has happened again and I'll try to hold the game session open for debugging purposes. So far I've traced function `play` down to `node.bufferSource.start(0, seek, duration)` in function `playWebAudio` which seems to be the normal path down to the native code. The `seek` and `duration` values look correct, and the `vol` value is correct as well. I compared the execution path of `play` to the execution path from a game session where the sound is working and the execution paths are exactly the same.

I'm not sure if something is set wrong in the buffer or context, or if the bug is in the native code. I'm not sure where to go from here. Any suggestions would be appreciated.
